Services/FileServices/classes/class.ilFileUtils.php

```diff
@@ -151,6 +151,7 @@ public static function rCopy(
                     $item->getPath(),
                     strlen($sourceDir)
                 );
+                self::makeDirParents(dirname(ILIAS_WEB_DIR . '/' . CLIENT_NAME .'/' . $itemPath));
```

Comment:
IMO this looks strange @mbecker-databay / @chfsx .

Why should we always create directories (recurisevely) within the `ILIAS_WEB_DIR`? What happens if `rCopy` is called in the context of the `ILIAS_DATA_DIR`?